### PR TITLE
Adds a tutorial button to the lobby menu

### PR DESCRIPTION
**Pull request by out-of-phaze**
## About The Pull Request
You can now play the tutorial from the lobby instead of having to spawn in as a ghost.

TODO:
- [x] Add confirmation prompt
- [x] Add Russian translation for confirmation prompt
- [x] Confirm Russian translation for "Tutorial"

## Why It's Good For The Game
It makes it easier for new players to actually find the tutorial instead of having to be linked a guide on how to find it, or luck into pressing an unlabeled button as an observer.

## Changelog
:cl: Penelope Haze
add: Added a tutorial button to the lobby menu.
/:cl:

### CODE REVIEW

File: code/__HELPERS/tegutranslate.dm [hunk 1]
```diff
@@ -98,6 +101,12 @@ GLOBAL_LIST_INIT(tegu_translation_list, list(
 	"Are you sure you wish to observe?" = list(
 		"Russian" = "Уверены что хотите наблюдать?",
 		),
+	"Are you sure you wish to play the tutorial?" = list(
+		"Russian" = "Вы уверенны что хотите пройти обучение?",
+		),
+	"Sorry, the tutorial is not available at this time! Try again later." = list(
+		"Russian" = "Извините, обучение недоступно сейчас, попробуйте позже.",
```
Comment (out-of-phaze) [inline code/__HELPERS/tegutranslate.dm:108]: Someone suggested `"Извините, обучение в данный момент недоступно, попробуйте позже."` for this instead. If that's preferable just say so and I can change it.

Comment (EgorDinamit) [reply code/__HELPERS/tegutranslate.dm:108]: Yes, that'd be more correct/better-looking translation. Could also do `"Извините, в данный момент обучение недоступно. Попробуйте позже."`.

Comment (EgorDinamit) [reply code/__HELPERS/tegutranslate.dm:108]: Yes, that'd be more correct/better-looking translation. Could also do `"Извините, в данный момент обучение недоступно. Попробуйте позже."`.

File: code/__HELPERS/tegutranslate.dm [hunk 2]
```diff
@@ -98,6 +101,12 @@ GLOBAL_LIST_INIT(tegu_translation_list, list(
 	"Are you sure you wish to observe?" = list(
 		"Russian" = "Уверены что хотите наблюдать?",
 		),
+	"Are you sure you wish to play the tutorial?" = list(
+		"Russian" = "Вы уверенны что хотите пройти обучение?",
```
Comment (EgorDinamit) [inline code/__HELPERS/tegutranslate.dm:105]:
```suggestion
		"Russian" = "Вы уверены что хотите пройти обучение?",
```

File: code/modules/mob/dead/new_player/new_player.dm [hunk 3]
```diff
@@ -74,6 +75,22 @@
 	popup.set_content(output.Join())
 	popup.open(FALSE)
 
+/mob/dead/new_player/proc/join_tutorial()
+	var/yes_text = TeguTranslate("Yes", src)
+	var/join_tutorial = alert(src, TeguTranslate("Are you sure you wish to play the tutorial?", src),"Player Setup",yes_text,TeguTranslate("No", src))
```
Comment (EgorDinamit) [inline code/modules/mob/dead/new_player/new_player.dm:80]:
```suggestion
	var/join_tutorial = alert(src, TeguTranslate("Are you sure you wish to play the tutorial?", src), "Player Setup", yes_text, TeguTranslate("No", src))
```
Not exactly a requirement, but it's preferred to add spaces between arguments.

**Review by dsmith328 — APPROVED** [review]
Works in game and helps greatly. Very nice!

**Pull request merged**